src/asm/parser.y

```diff
@@ -950,16 +953,43 @@ pushc		: T_POP_PUSHC	{ charmap_Push(); }
 popc		: T_POP_POPC	{ charmap_Pop(); }
 ;
 
-printt		: T_POP_PRINTT string	{ printf("%s", $2); }
+print		: T_POP_PRINT print_exprs
 ;
 
-printv		: T_POP_PRINTV const	{ printf("$%" PRIX32, $2); }
+println		: T_POP_PRINTLN { putchar('\n'); }
```

Comment:
Should we try to additionally `putchar('\r')` on Windows?

Reply:
Windows automatically prints out carriage returns when printing newlines in text mode.